Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -40,20 +51,33 @@ extension Style {
 
     /// Filters through source to determine supported locale styles.
     /// This is needed for v7 support
-    internal func getLocaleValue(locale: Locale) -> String {
+    internal func getLocaleValue(locale: Locale) -> String? {
+        // Reference for supported languages: https://docs.mapbox.com/ios/navigation/guides/localization-and-internationalization/#supported-languages
```

Comment:
this links into the docs for navigation. is maps limited to these languages as well?

Reply:
This is the only online docs that exist as a list of supported language.

Reply:
I thought the goal was just to link to some generic reference documentation in case someone wanted to look up the definition of one of these locales. https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html points to http://www.loc.gov/standards/iso639-2/php/English_list.php for language codes, https://www.iso.org/obp/ui/#search for country codes, and http://www.unicode.org/iso15924/iso15924-codes.html for the script designators. Perhaps just a link to the apple doc would be sufficient to help someone head in the right direction.

Reply:
But then they might pass in something that is not supported which would result in unexpected behavior. I think documenting based on guaranteed Mapbox support would be the better experience

Reply:
This is also an internal function

Reply:
> This is also an internal function

Exactly. I think the point of this documentation is to help someone look up the definition of one of the identifiers in the array, not provide a canonical list of supported locales.

Reply:
fair, that makes sense. Maybe our guides should have a list of supported languages